### PR TITLE
Dev/stretch

### PR DESCRIPTION
**Pull request by attiladoor**

### CODE REVIEW

File: debian/wb-configs.displace [hunk 1]
```diff
@@ -12,7 +12,7 @@
 /etc/hostapd.conf.wb
 /etc/default/hostapd.wb
 /etc/watchdog.conf.wb
-
+/etc/ssh/sshd_config
```
Comment (evgeny-boger) [inline debian/wb-configs.displace]: why no .wb extension? 
Why no systemd network config here?

File: configs-stretch/lib/systemd/system/networking.service.wb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[Unit]
```
Comment (evgeny-boger) [inline configs-stretch/lib/systemd/system/networking.service.wb]: what is this file needed for?

File: configs/etc/ssh/sshd_config [hunk 3]
```diff
@@ -0,0 +1,87 @@
+# Package generated configuration file
```
Comment (evgeny-boger) [inline configs/etc/ssh/sshd_config]: shouldn't it belong to config-stretch only?

File: debian/wb-configs.displace (hunk 1 above)
Comment (attiladoor) [reply debian/wb-configs.displace]: right, i add ".wb" to sshd_config.
network config is only for stretch and it is included in: https://github.com/contactless/wb-configs/blob/4d9a60cd8eedf5041584387370fd38c94f0c3885/debian/wb-configs-stretch.displace#L1

File: configs-stretch/lib/systemd/system/networking.service.wb (hunk 2 above)
Comment (attiladoor) [reply configs-stretch/lib/systemd/system/networking.service.wb]: https://contactless.atlassian.net/browse/SOFT-469

File: configs/etc/ssh/sshd_config (hunk 3 above)
Comment (attiladoor) [reply configs/etc/ssh/sshd_config]: it can be but it is actually the default config from wheezy. Should i move it to only stretch?

Comment (evgeny-boger) [reply configs/etc/ssh/sshd_config]: I think so. No need to substitute the config with the same file
